gef.py

```diff
@@ -2951,6 +2956,36 @@ def exit_handler(event):
         __gef_remote__ = None
     return
 
+def load_libc_args():
+    # load libc function arguments' definitions
+    if not get_gef_setting("context.libc_args"):
+        return
+
+    path = get_gef_setting("context.libc_args_path")
+    if path is None:
+        err("Settings context.libc_args_path not set but context.libc_args is True. Make sure you have gef-extras installed")
+        return
+
+    _arch_mode = current_arch.arch.lower() + '_' + current_arch.mode
+    _libc_args_file = "{}/{}.json".format(path, _arch_mode)
+
+    global libc_args_definitions
+
+    # current arch and mode already loaded
+    if _arch_mode in libc_args_definitions and len(libc_args_definitions[_arch_mode]):
```

Comment:
why are you checking the `len`?

Reply:
In the failure case below, maybe we can delete the key from the dictionary. Otherwise, it's fine to keep the len check.